com.unity.cinemachine/Runtime/Core/ConfinerOven.cs

```diff
@@ -236,16 +236,18 @@ public AspectStretcher(float aspect, float centerX)
 
         const long k_FloatToIntScaler = 100000;
         const float k_IntToFloatScaler = 1.0f / k_FloatToIntScaler;
-        const float k_MinStepSize = 0.005f;
+        
+        float m_MinStepSize = k_MinStepSize;
```

Comment:
should this be called m_StepSize or something like that?  It's a bit confusing between m_MinStepSize and k_MinStepSize.